modules/tensor_mechanics/src/materials/ComputeMeanThermalExpansionEigenstrainBase.C

```diff
@@ -17,13 +17,18 @@ ComputeMeanThermalExpansionEigenstrainBaseTempl<is_ad>::validParams()
   InputParameters params = ComputeThermalExpansionEigenstrainBaseTempl<is_ad>::validParams();
   params.addClassDescription("Base class for models that compute eigenstrain due to mean"
                              "thermal expansion as a function of temperature");
+  params.addParam<Real>("thermal_expansion_scale_factor",
```

Comment:
Should this be added to an "Advanced" parameter group?

Reply:
I am not sure what the requirements to consider it "advanced" are. But it does seem to make sense.

Reply:
Call it what you will. I just think it would be helpful to "tuck it away" somehow.